docs/resource-schemas.md

```diff
@@ -205,6 +215,7 @@ assets:
 | `externalDocsUrl` | Absolute URL to externally-hosted documentation.                                                                                                         | Optional | String           | No          | –             | –                                                                                                                                                                                                                                                                                                                                          |
 | `demoLinks`       | Links to demo sites. See [demo links](#demos-links).                                                                                                     | Optional | Array            | No          | –             | –                                                                                                                                                                                                                                                                                                                                          |
 | `noIndex`         | If set to `true`, the global catalogs will exclude the asset.                                                                                            | Optional | Boolean          | No          | `false`       | –                                                                                                                                                                                                                                                                                                                                          |
+| `docs`            | Contains information on where to find supporting documentation for the asset. See [asset docs](#asset-docs).                                             | Optional | Object           | No          | -             | –                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| `docs`            | Contains information on where to find supporting documentation for the asset. See [asset docs](#asset-docs).                                             | Optional | Object           | Yes          | -             | –                                                                                                                                                                                                                                                                                                                                          |
```

This should be inheritable. For example, the component docs for all of our common components will live in the Carbon Styles library. That's how the ~5 libraries like Carbon React, Carbon Angular, etc. will get their asset documentation.

That sets us up to better keep code and docs in lockstep going forward. For example, let's say a component gets a new feature in Carbon Styles (meaning there are new styles and updated documentation.) Carbon React picks up that new Carbon Styles version and implements the new feature so the Carbon React inherits that updated documentation that describes the new feature. If Carbon Angular hasn't upgraded Carbon Styles yet and don't implement that new feature yet, the Carbon Angular docs will properly not have that new feature documented.